src/foam/nanos/ruler/Rule.js

```diff
@@ -230,6 +234,48 @@
           ruler.getDelegate().cmd_(x.put("OBJ", obj), getCmd());
         }
       `
+    },
+    {
+      name: 'reverseAction',
```

Comment:
Should we move it to RuleAction instead? So that apply and reverse actions are grouped together instead of having the rule to configure them separately.

Reply:
You are right. I will move it to RuleAction.

We have no problems storing those rules because actions and predicates are modelled.